lib/fenrir_view/templates/template.js

```diff
@@ -1,18 +1,15 @@
-
-
-
-//----------------------------------------\\
+// ----------------------------------------\\
 // <%= pattern_type_name %>: <%= pattern_name %>
-//----------------------------------------\\
+// ----------------------------------------\\
 
-$(document).on('ready ajaxComplete', function() {
+// $(document).on('ready ajaxComplete', function() {
 
-  $('.<%= js_class_name %>').each(function() {
-    if ( !$(this).data('initialized') ) {
-      console.log("Setup events for a <%= pattern_name %>") // Your code goes here
+//   $('.<%= js_class_name %>').each(function() {
+//     if ( !$(this).data('initialized') ) {
+//       console.log("Setup events for a <%= pattern_name %>") // Your code goes here
 
-      $(this).data('initialized', true);
-    }
-  });
+//       $(this).data('initialized', true);
+//     }
+//   });
 
-});
+// });
```

Comment:
Did you mean to leave this commented out, or was this a temporary hack for dev?

If it's gone then delete it 💣